packages/open-next/src/wrappers/aws-lambda-streaming.ts

```diff
@@ -95,7 +98,18 @@ const handler: WrapperHandler = async (handler, converter) =>
 
       const response = await handler(internalEvent, streamCreator);
 
-      return converter.convertTo(response);
+      // If at this point the headers have not been written, it means that we are using the edge runtime
+      if (!headersWritten) {
```

Comment:
Actually this one is causing trouble. When using ISR/SSG `headersWritten` is false at this point (probably some kind of race condition as this is done on a stream )

`writeHeaders` below will throw an error, but somehow it still works for the user.